src/services/response_time/response_time.ts

```diff
@@ -0,0 +1,21 @@
+import { IService, Request, Response, Service } from "../../../mod.ts";
+
+export class ResponseTimeService extends Service implements IService {
+  #startTime = 0;
+
+  #endTime = 0;
+
+  constructor() {
+    super();
+  }
```

Comment:
Why constructor?

Reply:
No reason really, just didn’t realise until now how pointless it was :p good shout, I’ll remove it

Reply:
done :) https://github.com/drashland/drash/pull/567/commits/23be120b70398ca994ebd9d28f84cca4c704e7d7